unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -346,6 +346,12 @@
 
         If the width or height arguments are zero, it means the client should
         decide its own window dimension.
+
+        If this event is associated with a change to the properties of the
+        output, then it will be sent after the events announcing the property
+        changes have been sent. For example, if an output changes size, then
+        xdg_output.logical_size, ..., xdg_output.done would be sent before
```

Comment:
I'd rather not mention xdg-output here. layer-shell clients aren't supposed to configure themselves depending on the outputs logical size.